bin/goimports

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(pwd -P)"
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rootdir="$( cd "$bindir/.." && pwd )"
+targetbin="${rootdir}"/target/bin
+goimportsbin=${targetbin}/goimports
+
+# install goimports if it does not exist
+if [ ! -f "$goimportsbin" ]; then
```

Comment:
i think if we're going to do this check (which is great to avoid the download/installation hit), we should version the binary somehow, otherwise when we bump `golang.org/x/tools/cmd/goimports` in `go.mod`, this script won't pick it up.

if that's too much of a pain, we can just install every time.

```bash
$ ls -al target/bin
total 514560
drwxr-xr-x  9 siggy  staff       288 Aug 29 11:38 .
drwxr-xr-x  8 siggy  staff       256 Aug 21 15:34 ..
-rwxr-xr-x  1 siggy  staff  60235692 Jul 25 14:27 .docker-19.03.1
-rwxr-xr-x  1 siggy  staff  22921872 Aug 21 12:05 .golangci-lint-1.17.1
-rwxr-xr-x  1 siggy  staff  36022520 Aug 21 12:10 .kind-v0.4.0
-rwxr-xr-x  1 siggy  staff  38215872 Aug 29 11:39 .kind-v0.5.1
-rwxr-xr-x  1 siggy  staff  48591088 Aug 26 11:13 .kubectl-v1.15.3
-rwxr-xr-x  1 siggy  staff  47149640 Jul 30 09:34 helm-v2.14.3
-rwxr-xr-x  1 siggy  staff   9117784 Jun 15  2018 protoc-3.6.0
```